frontend/src/scenes/data-management/definition/definitionLogic.ts

```diff
@@ -85,6 +87,20 @@ export const definitionLogic = kea<definitionLogicType>([
 
                     return definition
                 },
+                deleteDefinition: async () => {
+                    if (values.isEvent) {
+                        await api.eventDefinitions.delete({ eventDefinitionId: values.definition.id })
+                    } else {
+                        await api.propertyDefinitions.delete({ propertyDefinitionId: values.definition.id })
```

Comment:
this works for person properties too?

Reply:
Yup. Well, property definitions are currently shared among events and persons.